examples/purchaseTesterTypescript/app/screens/OfferingDetailScreen.tsx

```diff
@@ -26,27 +25,21 @@ const OfferingDetailScreen: React.FC<Props> = ({ route, navigation }: Props) =>
   };
 
   const purchasePackage = (pkg: PurchasesPackage) => {
-    Purchases.purchasePackage(pkg).then((result) => {
+    Purchases.purchasePackage(pkg).then(() => {
```

Comment:
Interesting that we don't use the result... Maybe we should at least log it? But nothing to do in this PR.